products/pubsub/api.yaml

```diff
@@ -244,3 +244,44 @@ objects:
               If ttl is not set, the associated resource never expires.
               A duration in seconds with up to nine fractional digits, terminated by 's'.
               Example - "3.5s".
+      - !ruby/object:Api::Type::NestedObject
+        name: 'deadLetterPolicy'
+        allow_empty_object: true
```

Comment:
We probably don't want `allow_empty_object` here. It will allow us to specify `dead_letter_policy {}` in a config, which is invalid for this.

Reply:
Fixed.